.github/workflows/e2e-arm64.yaml

```
@@ -13,8 +13,6 @@ jobs:
         target:
           - linux-arm64-e2e
     steps:
-      - name: Collect Workflow Telemetry
-        uses: catchpoint/workflow-telemetry-action@v1
```

@jmhbnz, would it make sense to change it to run just when debug mode is active? i.e.,

```suggestion
      - name: Collect Workflow Telemetry
        uses: catchpoint/workflow-telemetry-action@v1
        if: runner.debug == '1'
```

Or is it not worth the effort/irrelevant?

---

This was an interesting feature to explore, I hadn't used it before.  I think for now we stick with removing them because it seems like the only way to trigger debug is to set `ACTIONS_STEP_DEBUG` to `true`  which sounds like a maintainer/org admin needs to be involved to set a variable or secret for the repo. 

If we could toggle the debug via k8s ci robot command I could see debug mode being quite handy but I don't think it can currently, we would need to write a new plugin.

---

You can enable it with the debug workflow/job button, still it does need to be triggered by a maintainer as you pointed it out

---

You can enable it with the debug workflow/job button, still it does need to be triggered by a maintainer as you pointed out.